README.md

```diff
@@ -9,6 +9,12 @@ Dart support for AppEngine).
 
 ## Local Development
 
+### Compile script.dart to script.dart.js
+
+```
+dart2js static/v2/js/script.dart -o static/v2/js/script.dart.js
```

Comment:
Run with `--minify --trust-primitives --trust-type-annotations`

From 310054 --> 106947

Reply:
Bummer, I had been running it with `-m` but removed it for DDC experiments.